### PR TITLE
Implement SC_CONTENTS_1~10

### PR DESCRIPTION
**Pull request by crazy-arashi**
* **Addressed Issue(s)**: None

* **Server Mode**: Both

* **Description of Pull Request**: 
Implemented SC_CONTENTS 1~10
Add item script for 102780,102781,102782 for renewal.

### CODE REVIEW

File: db/re/status.yml [hunk 1]
```diff
@@ -9045,3 +9045,93 @@ Body:
       NoClearance: true
       NoDispell: true
       NoRemoveOnDead: true
+  - Status: Contents_1
+    Icon: EFST_CONTENTS_1
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9051]: Status only affects SCB_BASE
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 2]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_2
+    Icon: EFST_CONTENTS_2
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9060]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 3]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_3
+    Icon: EFST_CONTENTS_3
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9069]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 4]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_4
+    Icon: EFST_CONTENTS_4
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9078]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 5]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_5
+    Icon: EFST_CONTENTS_5
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9087]:
```suggestion
      Aspd: true
      Base: true
```

File: db/re/status.yml [hunk 6]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_6
+    Icon: EFST_CONTENTS_6
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9096]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 7]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_7
+    Icon: EFST_CONTENTS_7
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9105]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 8]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_8
+    Icon: EFST_CONTENTS_8
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9114]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 9]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_9
+    Icon: EFST_CONTENTS_9
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9123]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 10]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
+  - Status: Contents_10
+    Icon: EFST_CONTENTS_10
+    CalcFlags:
+      All: true
```
Comment (Atemo) [inline db/re/status.yml:9132]:
```suggestion
      Base: true
```

File: db/re/status.yml [hunk 11]
```diff
@@ -9045,3 +9045,93 @@ Body:
...
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
```
Comment (Atemo) [inline db/re/status.yml:9137]:
```suggestion
      NoClearance: true

```

File: db/re/item_db_usable.yml [hunk 12]
```diff
@@ -78155,6 +78155,8 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+    Script: |
+      sc_start SC_CONTENTS_1,900000,10;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:78159]: There are specialeffect on player according to https://www.divine-pride.net/database/item/102780 (same effects on my data)
```suggestion
      sc_start SC_CONTENTS_1,900000,10;
      specialeffect2 EF_POTION_BERSERK;
      specialeffect2 EF_PROVIDENCE;
```

File: db/re/item_db_usable.yml [hunk 13]
```diff
@@ -78168,6 +78170,8 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+    Script: |
+      sc_start SC_CONTENTS_2,900000,10;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:78174]: According to https://www.divine-pride.net/database/item/102781
```suggestion
      sc_start SC_CONTENTS_2,900000,10;
      specialeffect2 EF_POTION_BERSERK;
      specialeffect2 EF_SPELLBREAKER;
```

File: db/re/item_db_usable.yml [hunk 14]
```diff
@@ -78181,6 +78185,8 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+    Script: |
+      sc_start SC_CONTENTS_3,900000,15;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:78189]: According to https://www.divine-pride.net/database/item/102782
```suggestion
      sc_start SC_CONTENTS_3,900000,15;
      specialeffect2 EF_POTION_BERSERK;
      specialeffect2 EF_HASTEUP;
```

**Review by Atemo — APPROVED** [review]

**Review by aleos89 — APPROVED** [review]

**Pull request merged**